### ISSUE TITLE
Keeps writing in bold after line break

### ISSUE THREAD

**Issue by natarius** [nylas/N1#964]
N1's email editor seems to be inspired by Medium...which is awesome!

But medium switches back to regular font weight after a line break were N1 keeps the weight bold! I wish you would mirror medium's behavior.

Thanks

**Comment by grinich** [comment]
I'm going to close this as stale. Please open a new issue if this is still occurring.

**Comment by natarius** [comment]
Still happening :(  

  

Sent from [Nylas N1](https://nylas.com/n1?ref=n1), the extensible, open source
mail client.  

On Apr 27 2016, at 12:12 pm, Michael Grinich &lt;notifications@github.com&gt;
wrote:  

> I'm going to close this as stale. Please open a new issue if this is still
occurring.

>

> —  
You are receiving this because you authored the thread.  
Reply to this email directly or [view it on GitHub](https://github.com/nylas/N
1/issues/964#issuecomment-215197151)![](https://github.com/notifications/beaco
n/AAlEY_JJqKkfFXhFX89uANXIJcgBATFIks5p77UrgaJpZM4HCoMM.gif)

**Comment by grinich** [comment]
This is a pretty tricky thing to figure out, since editors like MS Word will keep the bold setting.

**Comment by natarius** [comment]
True. But since you are aiming for a medium like experience (and the hover menu hints at that) it's broken :/

I also can't think of many instances where I would want to write or read a email with paragraphs of bold text

—
